OutOfSchool/OutOfSchool.WebApi.Tests/Redis/CacheServiceTests.cs

```diff
@@ -85,14 +86,71 @@ public async Task GetOrAddAsync_WhenDataNotExistsOrExpired_ShouldSaveNewData()
     }
 
     [Test]
-    public async Task RemoveAsync_ShouldCallCacheRemove()
+    public async Task RemoveAsync_ShouldCallCacheRemoveOnce()
     {
         // Arrange & Act
         await cacheService.RemoveAsync("Example");
 
         // Assert
         distributedCacheMock.Verify(
-            c => c.RemoveAsync("Example", It.IsAny<CancellationToken>()),
+            c => c.Remove("Example"),
+            Times.Once);
+    }
+
+    [Test]
+    public async Task ReadAsync_WhenDataExistsInCacheAndNotExpired_ShouldReturnData()
+    {
+        // Arrange
+        var expected = new Dictionary<string, string>()
+        {
+            {"ExpectedKey", "ExpectedValue"},
+        };
```

Comment:
(Complicated test) Please pay attention what you are testing.

For this specific test (and few next) - sorry, but `expected` variable:
1. It is not "expected" (expected is serialized presentation of this dictionary)
2. It has wrong type - as `ReadAsync` returns just string without extra processing - why not check strings without that serialization / Contains magic?
3. It is hardcoded - why not use Bogus random string?

Reply:
Ok, done.
1) Used Bogus random string for generating data.
2) Removed dictionary.